### ISSUE TITLE
consider migrating to bootstrap 3

### ISSUE THREAD

**Issue by alexrudnick** [hltdi/guampa#25]
They went and released a new major version of bootstrap on us!

http://blog.getbootstrap.com/2013/08/19/bootstrap-3-released/

http://bootply.com/bootstrap-3-migration-guide

**Comment by alexrudnick** [comment]
This is pretty low priority. Make everything work right first.

**Comment by taylskid** [comment]
Yeah, I noticed that there were some pretty significant changes between 2.x and 3. I'm also pretty sure 3 breaks compatibility with some portions of IE8 and iirc, IE7 and below.
